### PR TITLE
Add mysql to travis

### PR DESCRIPTION
**Pull request by andrewgy8**
**Problem**
What problem have you solved?

We have mysql specific testing, but nothing in the CI

**Solution**
How did you solve the problem?

Added mysql to the CI pipeline

**Acceptance Criteria**
Please make sure you meet the following.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] New and existing unit tests pass on Travis with my changes

### CODE REVIEW

**Comment by bmihelac** [comment]
I see errors from travis:

```
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/MySQLdb/connections.py", line 224, in query
    _mysql.connection.query(self, query)
django.db.utils.OperationalError: (1366, "Incorrect string value: '\\xC5\\x88\\xC3\\xAD\\xC4\\x8D...' for column 'name' at row 1")
```
It looks like MySql error related to unicode characters. I think adding charset and collation to mysql database setting in tests/settings.py would help

```
            'CHARSET': 'utf8',
            'COLLATION': 'utf8_general_ci',
```

**Comment by andrewgy8** [comment]
@bmihelac Thanks for the suggestion.  I seem to have gotten it working with an explicit `.encode()` on the string that was causing issues. 

Although, I have never worked with MySQL before, I believe this is a MySQL issue.  Have you worked with it before, or do you know if this is something that needs to be done?

File: tests/core/tests/test_widgets.py [hunk 1]
```diff
@@ -217,7 +217,7 @@ class ManyToManyWidget(TestCase):
     def setUp(self):
         self.widget = widgets.ManyToManyWidget(Category)
         self.widget_name = widgets.ManyToManyWidget(Category, field="name")
-        self.cat1 = Category.objects.create(name='Cat úňíčóďě')
+        self.cat1 = Category.objects.create(name='Cat úňíčóďě'.encode())
```
Comment (bmihelac) [inline tests/core/tests/test_widgets.py]: There is no need for `encode` here

**Comment by bmihelac** [comment]
@andrewgy8 I have added two comments. If CI fails even after this changes, I'll check it out.

File: tests/core/tests/test_widgets.py (hunk 1 above)
Comment (andrewgy8) [reply tests/core/tests/test_widgets.py]: Yeah I thought that too. However, after reading and experimenting a bit, it seems that is the only way to get the tests passing. 😟 

Ill try what you suggested above and see if it works.

Comment (bmihelac) [reply tests/core/tests/test_widgets.py]: It look like it is still failing. I'll take a look at it as soon as I catch a little time today

Comment (andrewgy8) [reply tests/core/tests/test_widgets.py]: No worries. The encoding seems to make it work. So either its a setting issue in MySQL which needs configuring, or less likely, stings need `encode` explicitly.

Comment (andrewgy8) [reply tests/core/tests/test_widgets.py]: And FYI, you can run it locally with ` docker run -e MYSQL_ROOT_PASSWORD=password -d -p 3306:3306 mysql` and then `IMPORT_EXPORT_TEST_TYPE=mysql-innodb ./runtests.sh`

Comment (bmihelac) [reply tests/core/tests/test_widgets.py]: ok, I see that test database had default collation `latin1_swedish_ci`. 

I have added commit and squased in:
https://github.com/django-import-export/django-import-export/pull/905

Comment (andrewgy8) [reply tests/core/tests/test_widgets.py]: `latin1_swedish_ci`? 😕 

Any idea why? Is that a travis thing?

Comment (bmihelac) [reply tests/core/tests/test_widgets.py]: no, `latin1_swedish_ci`  is default :)

Comment (andrewgy8) [reply tests/core/tests/test_widgets.py]: Clearly this makes sense only because I'm swedish! 😆

**Comment by andrewgy8** [comment]
Closing in favor of #915

**Pull request closed without merging**